python/ftdi2.py

```diff
@@ -228,7 +228,7 @@ def get_device_info_detail(dev=0):
     lpdwLocId = c.c_ulong()
     pcSerialNumber = c.c_buffer(MAX_DESCRIPTION_SIZE)
     pcDescription = c.c_buffer(MAX_DESCRIPTION_SIZE)
-    ftHandle = c.c_ulong()
+    ftHandle = c.c_void_p()
```

Comment:
If we do this, we must make sure that falsy checks are valid because a null c_void_p translates to None, not 0. I think the following is more safe.
```suggestion
    ftHandle = c.c_size_t()
```

Reply:
If that works, I’m fine with it. Where are the falsy checks happening? I couldn’t find any in this file.